### PR TITLE
Add stale action for close the inactive issues/prs

### PR DESCRIPTION
**Pull request by chenrui333**
Add github action for actively close inactive issues or PRs.

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GH_ACTION_STALE_KEY }}
+        stale-issue-message: >
+          This issue is stale because it has been open 30 days with no activity.
+          Remove stale label or comment or this will be closed in 5 days
+        days-before-stale: >
```
Comment (SMillerDev) [inline .github/workflows/stale.yml]: Isn't this a duplicate key?

**Review by SMillerDev — COMMENTED** [review]
I think this should be in the shared repo between all projects right @Homebrew/maintainers ?

**Comment by reitermarkus** [comment]
We already have https://github.com/Homebrew/.github/blob/master/stale.yml. Does this not apply to this repo?

**Comment by issyl0** [comment]
As far as I’ve seen the stale bot hasn’t worked for some time. Maybe we should fix that rather than adding more code?

**Comment by SMillerDev** [comment]
> We already have https://github.com/Homebrew/.github/blob/master/stale.yml. Does this not apply to this repo?

I'd say it would. But as issy says, I haven't seen it work for a while

**Comment by reitermarkus** [comment]
It was enabled only for `linuxbrew-core` for some reason. I set it to “All repositories” again.

**Comment by chenrui333** [comment]
@issyl0, from what I can stale.yml would go away in the future, and github action would stay. But I am happy to just modify staley.yml. :)

**Comment by chenrui333** [comment]
> https://github.com/Homebrew/.github

Never saw such thing before.

**Comment by chenrui333** [comment]
As it has been enabled for all repos, closing my PR now

**Pull request closed without merging**